dart/example/main.dart

```diff
@@ -48,7 +52,9 @@ Future<void> main() async {
       ..setTag('build', '579')
       ..setExtra('company-name', 'Dart Inc');
   });
+}
 
+void runApp() async {
```

Comment:
```suggestion
Future<void> runApp() async {
```

Reply:
I'm not sure about it, the function will never return something.

Reply:
+ the callback is not awaited
```dart
runZonedGuarded(() {
      callback();
    }, (exception, stackTrace) async {//...}
})
```

Reply:
yeah not really needed

Reply:
according to https://dart.dev/codelabs/async-await

`Future<void> main() async`

even a `main` method has a `Future<void>` if its body is `async`, so wondering if we should do the same.

Reply:
I'm quite convicend that a `FutureOr<void>` def. for `callback` is the safest way, also:
`await callback();`
worst case, it's not used at all as most of the people will just do `void`, but if they wanna do async stuff, we don't need a breaking change again, wdyt?

Reply:
✅ I changed it to `FutureOr<void>` but I'm not sure of what is the safest way https://github.com/dart-lang/sdk/issues/33218